app/views/devise/invitations/edit.html.erb

```diff
@@ -31,7 +31,7 @@
     <div class="registration__field">
       <%= f.form_row :password_confirmation do %>
         <%= f.errors_for :password_confirmation %>
-        <%= f.label :password_confirmation, class: "form__label-heading" %>
+        <%= f.label :password_confirmation, t('authentication.accept_invitation_page.field_labels.password_confirmation'), class: "form__label-heading" %>
```

Comment:
Ah, I see. I was just following the pattern that was already here. If I was to use that method of doing the password confirmation here then I'd be introducing another way of doing the locale lookup, which I was keen not to do in this PR (the rabbit hole was deep enough) :)